docs/nodes/credentials/Sendy/README.md

```diff
@@ -0,0 +1,20 @@
+---
+permalink: /credentials/sendy
+description: Learn to configure credentials for the Sendy node in n8n
+---
+
+# Sendy
+
+You can use these credentials to authenticate the following nodes with Sendy.
+- [Sendy](../../nodes-library/nodes/Sendy/README.md)
+
+## Prerequisites
+
+Host a [Sendy](https://sendy.co/get-started) application.
+
+## Using API Key
+
+1. Access your Sendy application.
+2. Click on the username on the top right corner and select 'Settings'.
+3. Copy the API key displayed under the ***Your API key*** section.
+4. Use this API key and the installation URL with your Sendy node credentials in n8n.
```

Comment:
<img width="69" alt="Screenshot 2020-10-22 at 09 22 18" src="https://user-images.githubusercontent.com/7481165/96838362-17893180-1448-11eb-95f7-f1bbee51cecf.png">

Reply:
@tanay1337 this is how it is displayed in Sendy. I am not sure if I am missing something. 

<img width="289" alt="Screenshot 2020-10-22 at 3 54 51 PM" src="https://user-images.githubusercontent.com/18901032/96859427-1f17fd00-147f-11eb-953a-210d59c7d06c.png">

Reply:
Ah okay, never mind then!